tests/codegen-llvm/dynamic_shared_memory.rs

```diff
@@ -0,0 +1,27 @@
+// Checks that dynamic_shared_memory works.
+
+//@ revisions: amdgpu nvptx x86
+//@ compile-flags: --crate-type=rlib
+//
+//@ [amdgpu] compile-flags: --target amdgcn-amd-amdhsa -Ctarget-cpu=gfx900
+//@ [amdgpu] only-amdgpu
+//@ [amdgpu] needs-llvm-components: amdgpu
+//@ [nvptx] compile-flags: --target nvptx64-nvidia-cuda
+//@ [nvptx] only-nvptx64
+//@ [nvptx] needs-llvm-components: nvptx
+//@ [x86] compile-flags: --target x86_64-unknown-linux-gnu
+//@ [x86] only-x86_64
```

Comment:
This is not enough when running on x86_64 Windows. You did need `only-x86_64-unknown-linux-gnu` I think.